diff --git a/dist/renderers/RenderBlock.svelte b/dist/renderers/RenderBlock.svelte
index 2b817a73570e5ea66a74939fefdac850dff63c41..f2ee732b29f441a7331ca331c18891d194a7e6d8 100644
--- a/dist/renderers/RenderBlock.svelte	
+++ b/dist/renderers/RenderBlock.svelte	
@@ -12,9 +12,9 @@
 
   let {global, node, indexInParent, children}: RenderBlockProps = $props()
 
-  let {components} = $derived(global)
-  let {style = 'normal'} = $derived(node)
-  let blockComponent = $derived(
+  const components = $derived(global.components)
+  const style = $derived(node.style ?? 'normal')
+  const blockComponent = $derived(
     typeof components.block === 'function' ? components.block : components.block[style]
   )
   $effect(() => {
@@ -23,7 +23,7 @@
     }
   })
 
-  let blockProps = $derived.by(() => {
+  const blockProps = $derived.by(() => {
     return {
       global,
       indexInParent,
@@ -31,7 +31,7 @@
     } as BlockComponentProps
   })
 
-  let BlockComponent = $derived(blockComponent || components.unknownBlockStyle)
+  const BlockComponent = $derived(blockComponent || components.unknownBlockStyle)
 </script>
 
 <BlockComponent portableText={blockProps}>
diff --git a/dist/renderers/RenderCustomBlock.svelte b/dist/renderers/RenderCustomBlock.svelte
index e27ce766df85e6bccebe6de4d45ee570c04d63d3..f3c5744526d6f320283c15096ca00af2b358035b 100644
--- a/dist/renderers/RenderCustomBlock.svelte	
+++ b/dist/renderers/RenderCustomBlock.svelte	
@@ -18,9 +18,9 @@
     isInline = false
   }: RenderCustomBlockProps = $props()
 
-  let {components} = $derived(global)
-  let {_type} = $derived(node)
-  let customComponent = $derived(components.types[_type])
+  const components = $derived(global.components)
+  const _type = $derived(node._type)
+  const customComponent = $derived(components.types[_type])
 
   $effect(() => {
     if (!customComponent) {
@@ -40,7 +40,7 @@
     })()
   )
 
-  let CustomComponent = $derived(customComponent || components.unknownType)
+  const CustomComponent = $derived(customComponent || components.unknownType)
 </script>
 
 <CustomComponent portableText={componentProps} />
